### ISSUE TITLE
Language

### ISSUE THREAD

**Issue by john-goes** [john-goes/fishthephishing#3]
Does it make sense to have the name of the project (Fish the Phishing) in English and all the rest in Portuguese (as we want to reach a Brazilian public) or make everything in English and publish to all over the world in English. 
Or both?

Nowadays it is everything in Portuguese and just the name is English. It's in Portuguese because the public we wanted to reach is Stone Co.'s, however it would definitely be available to the general public sooner or later.

TL;DR: 
Should we just translate everything to English and stop using Portuguese to reach a larger public?
